Cargo.toml

```diff
@@ -27,8 +28,13 @@ quickcheck = { version = "0.9.2", optional = true }
 rand = { version = "0.7.3", optional = true }
 serde = { version = "1.0.116", default-features = false, optional = true }
 serde_bytes = { version = "0.11.5", optional = true }
+arbitrary = { version = "1.1.0", optional = true , features = ["derive"] }
 
 core2 = { version = "0.4", default-features = false }
 
 [dev-dependencies]
 serde_json = "1.0.59"
+rand = "0.7.3"
+arbitrary = "1.1.0"
+quickcheck = "0.9.2"
+multihash = { git = "https://github.com/multiformats/rust-multihash", branch = "master", features = ["arb"]}
```

Comment:
Why do you need to add those as dev dependencies. It should be enough to have them in the dependencies already.

Reply:
Because features don’t enable during test builds, and I added a test into the arb.rs file. So it finds the test in that file and then tries to build it without arb feature.

Reply:
Looking at it, the whole `src/arb.rs` only makes sense with the `arb` features. So we could put a `#![cfg(feature = "arb")]` at the top of the file and it would only be compiled if the `arb` feature is enabled.